site_config

```diff
@@ -0,0 +1,23 @@
+server {
+        listen 80;
+        listen [::]:80;
+        root /var/www/site1;
+
+        # Add index.php to the list if you are using PHP
+        index index.php index.html index.htm index.nginx-debian.html;
+
+        server_name site1.com;
+
+        location / {
+                try_files $uri $uri/ =404;
+        }
+
+        location ~ \.php$ {
+                include snippets/fastcgi-php.conf;
+        	     # With php-fpm (or other unix sockets):
+                fastcgi_pass unix:/run/php/php8.1-fpm.sock;
```

Comment:
Please, mention the purpose of these lines.

Reply:
> Please, mention the purpose of these lines.
Line 16: fastcgi-php.conf file contains the code to include fastcgi.conf file which contains the configuration settings of the FPM
Line 18: Nginx is sending request to php-fpm through a unix socket (i.e communicating between processes in the same system). The request is sent at the location /run/php/php8.1-fpm.sock;